app/assets/stylesheets/earthworks.css

```diff
@@ -206,12 +206,13 @@ Uncomment lines 142-144
 
 /* Styles for search result description expand/collapse */
 
-.expand-collapse .btn-outline-secondary {
+.expand-collapse .btn-outline-primary {
   display: flex;
-  &.active {
-    color: white;
-    background-color:var(--stanford-cardinal);
-  }
+  --bs-btn-hover-color: var(--stanford-digital-blue-dark);
```

Comment:
None of these are necessary anymore, right? Isn't this the default?

Reply:
See https://github.com/sul-dlss/component-library/blob/355885ec11bcaca6d601bb7eae3b44251e44fa0d/styles/button.css#L37

Reply:
I just reviewed, and you are right that several are default in the component library and can be removed.  But the active button turning solid blue isn't the default and I think it's what is requested.  So I think a couple overrides are still relevant.

Reply:
Actually, was able to adjust the existing `&.active` definition to be blue instead of red.  Seemed to do it.